docs/compatibility.rst

```diff
@@ -0,0 +1,98 @@
+.. _compatibility:
+
+=============
+Compatibility
+=============
+
+.. rubric:: Table of Contents
+
+.. contents::
+   :local:
+
+Doctrine
+========
+
+Unsupported Features
+--------------------
+
+The following Doctrine features are not supported:
+
+- Fulltext indexes
+
+- `DQL joins`_
+
+- Joins
+
+  - However, referencing relations can be done without joins using Doctrine's
+    lazy loading mechanism with subsequent SELECTs (except many-to-many
+    releations)
+
+Column Definitions
+------------------
+
+Unfortunately, when defining CrateDB `object`_ or `array`_ type columns, you
```

Comment:
Primitive column types, such as string, integer, ... can be defined in the regular Doctrine way.